experimental/PropertyDDS/packages/property-changeset/.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         "@typescript-eslint/no-unused-expressions": "off",
         "@typescript-eslint/no-var-requires": "off",
         "@typescript-eslint/prefer-for-of": "off",
+        "@typescript-eslint/prefer-includes": "off",
```

Comment:
Should we leave a comment explaining why we're disabling this here / what (if any) follow-up work there is to do to enable it?

Reply:
I added a comment pointing to #10316.